service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/auth/IotHubServiceSasToken.java

```diff
@@ -20,7 +20,7 @@
  */
 public final class IotHubServiceSasToken
 {
-    long TOKEN_VALID_SECS = 365*24*60*60;
```

Comment:
I want to call this out as a change that I believe isn't a breaking change, but I'd like to hear some opinions on this.

With CBS auth and proactive token renewal, customers who were used to having connection stay open for a long time (more than an hour) will still see the same behavior even though the SDK will be proactively renewing their connection with shorter lived tokens rather than opening the connection with one really long-lived SAS token

Reply:
Seems fine to me. Can you make your best case as to why it would not be fine?

Reply:
If the cbs roken renewal logic breaks for any reason, users will see earlier disconnections than expected. I don't see it as a very likely issue, though. I plan on adding some token renewal tests for covering that scenario

Reply:
lgtm

Reply:
I'd say that is a good thing.